packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -44,240 +53,184 @@ export default {
-    });
-  },
-  decorators: [
-    (story: () => typeof html) => html`
-      <style>
-        td.template {
-          width: 100%;
-          height: 250px;
-        }
-      </style>
-      ${story()}
-    `
-  ]
-};
-
-/**
- * Use the not-interactive attribute to create a non-interactive step group.
- */
-
-export const notInteractive = {
-  name: 'Not Interactive X Size',
-  parameters: { controls: { exclude: 'not-interactive' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'not-interactive' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  },
-  decorators: [
-    (story: () => typeof html) => html`
-      <style>
-        td.template {
-          width: 100%;
-        }
-      </style>
-      ${story()}
-    `
-  ]
-};
+  name: 'Orientation',
+  render: () => html`
+    <div class="space-y-5">
+      <sd-step-group size="lg" orientation="horizontal" active-step="1">
+        <sd-step size="lg" orientation="horizontal" state="default">
+          <p slot="label">Label</p>
+        </sd-step>
 
-/**
- * Use the 'base' and 'body' parts to style the step-group.
- */
-export const Parts = {
-  parameters: {
-    controls: {
-      exclude: ['base', 'body']
-    }
-  },
-  render: () => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'orientation' },
-        y: {
-          type: 'template',
-          name: 'sd-step-group::part(...){outline: solid 2px red}',
-          values: ['base', 'body'].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-step-group::part(${part}){outline: solid 2px red; outline-offset: 0px;}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      args: overrideArgs([
-        {
-          type: 'slot',
-          name: 'default',
-          value: `
-            <sd-step size="lg" orientation="horizontal" state="default">
-              <p slot="label">Lorem ipsum dolor sit</p>
-              Lorem ipsum est dolor sit amet
-            </sd-step>
-    
-            <sd-step size="lg" orientation="horizontal" state="current">
-              <p slot="label">Exercitation ullamco laboris</p>
-              Lorem ipsum est dolor sit amet
-            </sd-step>
-    
-            <sd-step size="lg" orientation="horizontal" state="disabled">
-              <p slot="label">Reprehenderit qui in e name</p>
-              Lorem ipsum est dolor sit amet
-            </sd-step>`
-        },
-        {
-          type: 'attribute',
-          name: 'active-step',
-          value: `1`
-        }
-      ])
-    });
-  }
-};
+        <sd-step size="lg" orientation="horizontal" state="current">
+          <p slot="label">Label</p>
+        </sd-step>
 
-/**
- * sd-steps are fully accessibile via keyboard.
- */
+        <sd-step size="lg" orientation="horizontal" state="disabled">
```

Comment:
For size only show horizontal orientation:

![image](https://github.com/user-attachments/assets/340b1620-e5c7-4bbf-a6cb-486db028cf03)

Reply:
Updated